app/controllers/merchant/items_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Merchant::ItemsController < Merchant::BaseController
+
+  def index
+    @merchant = Merchant.first
+  end
+
+  def update
+    item = Item.find(params[:id])
+    item.toggle!(:active?)
+    if item.active?
```

Comment:
Here you could add conditional for the path, ie. if params[:activate_deactivate] == 'activate'